api/src/test/java/uk/gov/dhsc/htbhf/claimant/converter/AddressDTOToAddressConverterTest.java

```diff
@@ -28,6 +30,18 @@ void shouldConvertAddressDTOToEquivalentAddressObject() {
         assertThat(result.getPostcode()).isEqualTo(addressDTO.getPostcode());
     }
 
+    @Test
+    void shouldConvertPostcodeToUppercase() {
+        // Given
+        AddressDTO addressDTO = anAddressDTOWithPostcode("aa11aa");
+
+        // When
+        Address result = converter.convert(addressDTO);
+
+        // Then
+        assertThat(result.getPostcode()).isEqualTo(addressDTO.getPostcode().toUpperCase());
```

Comment:
I'd prefer this to specifically say "AA11AA" as the expected value rather than take it from the DTO.